static/js/constants.js

```diff
@@ -209,6 +209,8 @@ export const NEW_APPLICATION = "newApplication"
 export const TAKE_VIDEO_INTERVIEW = "TAKE_VIDEO_INTERVIEW"
 export const RESUME_UPLOAD = "resumeUpload"
 
+export const MAX_RESUME_FILE_SIZE = 15000000 // 15MB
```

Comment:
The limit on the nginx/server side seems to be lower than this, when I try uploading a 10 MB file it fails with a generic `Your file failed to upload` message.  Do you think it might be worth setting this as an .env variable that gets passed to the front end to make it more configurable?

Reply:
@mbertrand Great idea